tests/cupy_tests/core_tests/fusion_tests/test_function.py

```diff
@@ -141,7 +141,7 @@ def func(x):
     @fusion_utils.check_fusion(generate_inputs_name='dtype_combination')
     def test_reduction_pairwise_premap(self, xp, dtype1, dtype2):
         def func(x, y):
-            return xp.sum(xp.sqrt(x - y))
```

Comment:
Is this test newly introduced in this PR?

Reply:
Yes.